client/src/components/NavBar.jsx

```diff
@@ -62,7 +62,7 @@ const NavBar = ({ className }) => {
     const fullscreenPages = ['/founduser'];
 
     const hideNavbar = useMemo(
-        () => fullscreenPages.includes(location.pathname) && !app.isSearching,
+        () => fullscreenPages.includes(location.pathname) && app.currentChatId,
```

Comment:
what's the reason of this change here?

Reply:
Only show the navbar when we're not currently chatting.

The previous one works, but only when searching for a new buddy. The new change makes it available for the alert screen as well

Reply:
please remove that and create a new issue for it with it's solution

Reply:
It's also part of the feature imo. Is there a special reason for creating a whole new issue for it?

Reply:
It's not, please create a new issue

Reply:
How is it not? Can you try removing that line to see why it's the same feature?

Reply:
I see, my mistake for the misunderstanding